snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -338,9 +338,6 @@ func (b *bootstrapper) Start(ctx context.Context, startReqID uint32) error {
 		if err != nil {
 			return fmt.Errorf("failed to create stop vertex: %w", err)
 		}
-		if err := stopVertex.Verify(ctx); err != nil {
-			return fmt.Errorf("failed to verify stop vertex: %w", err)
-		}
```

Comment:
Unlike block or tx verify, vertex verification was stateless, so this didn't do anything